modules/ROOT/pages/storage.adoc

```diff
@@ -15,9 +15,9 @@ Currently, if the root filesystem is smaller than 8 GiB, a warning is emitted on
 
 Many of the examples below will reference a block device, such as `/dev/vda`.  The name of the available block devices depends on the underlying infrastructure (bare metal vs cloud), and often the specific instance type.  For example in AWS, some instance types have NVMe drives (`/dev/nvme*`), others use `/dev/xvda*`.
 
-You will need to look for relevant documentation for your infrastructure around this.
+In most cases, the simplest way to refer the target block device is using the `/dev/disk/by-id/coreos-boot-disk` link, which is created automatically and available only during machine provisioning.
```

Comment:
I think maybe we should add a little more about what this link is. Maybe something like:

```suggestion
If your disk configuration is simple and uses the same disk the OS was booted from then the `/dev/disk/by-id/coreos-boot-disk` link can be used to conveniently refer to that device. This link is only available during provisioning for the purpose of making it easy to refer to the same disk the OS was booted from.
```